src/jit/liveness.cpp

```diff
@@ -1103,6 +1103,13 @@ void Compiler::fgExtendDbgLifetimes()
             {
                 continue;
             }
+#if !defined(_TARGET_64BIT_) && !defined(LEGACY_BACKEND)
+            // On x86, this applies to long-typed variables as well once we've hit LIR. 
```

Comment:
We run liveness after Rationalizer, but before Decomposition, so I don't think you can skip these.

---
In reply to: [79941261](https://github.com/dotnet/coreclr/pull/7294#discussion_r79941261) [](ancestors = 79941261)